experimental/traffic-portal/src/app/core/certs/cert-detail/cert-detail.component.scss

```diff
@@ -0,0 +1,34 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+div.mat-elevation-z2 {
```

Comment:
`mat-elevation-z2` sounds like an Angular Material class - if that's the case, I'd suggest adding a new class rather than extending it here, because in any other context the exact same class would apply a different styling. If that's not the case, I suggest changing the name so that people reading it don't think there's some other styling not defined here being applied.